guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -1,35 +1,32 @@
 [id="Enrolling_Server_with_the_AD_Server_{context}"]
-= Enrolling {ProjectServer} with the AD server
+= Enrolling {ProjectServer} with the Active Directory server
 
-In the {Project} CLI, enroll {ProjectServer} with the Active Directory server.
+In the {Project} CLI, enroll {ProjectServer} with the Active Directory (AD) server.
 
-.Prerequisites
-* GSS-proxy and {nfs-client-package} are installed.
-+
-Installing GSS-proxy and {nfs-client-package}:
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-sssd_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using SSSD].
```

Comment:
Actually, let's try to drop the whole module and mention the requirement for the Foreman server to be a member of the AD domain as a prerequisite in the next section. I am keeping the link for all builds because SSSD doesn't have comprehensive user documentation upstream.

The whole workflow looks much simpler now, which is what we want.